ddtrace/contrib/redis/patch.py

```diff
@@ -9,7 +9,7 @@
 from .util import format_command_args, _extract_conn_tags
 
 
-def patch():
+def patch(tracer=None):
```

Comment:
if we override the default tracer (``ddtrace.tracer``), we may avoid passing the ``tracer`` around our instrumentation code. As suggested, we can simply:
```python
import ddtrace

# at some point
tracer = ddtrace.tracer
```